debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MysqlDefaultValueIT.java

```diff
@@ -525,7 +526,7 @@ public void realTest() throws InterruptedException {
     public void numericAndDecimalToDoubleTest() throws InterruptedException {
         config = DATABASE.defaultConfig()
                 .with(MySqlConnectorConfig.SNAPSHOT_MODE, MySqlConnectorConfig.SnapshotMode.INITIAL)
-                .with(MySqlConnectorConfig.DECIMAL_HANDLING_MODE, JdbcValueConverters.DecimalMode.DOUBLE)
+                .with(MySqlConnectorConfig.DECIMAL_HANDLING_MODE, RelationalDatabaseConnectorConfig.DecimalHandlingMode.DOUBLE)
```

Comment:
isn't this the correct config value?
is it okay to change the parameter with that PR?

Reply:
Yes, that's ok